compiler/rustc_parse/src/parser/ty.rs

```diff
@@ -397,10 +397,14 @@ impl<'a> Parser<'a> {
     fn parse_ty_ptr(&mut self) -> PResult<'a, TyKind> {
         let mutbl = self.parse_const_or_mut().unwrap_or_else(|| {
             let span = self.prev_token.span;
-            let msg = "expected mut or const in raw pointer type";
+            let msg = "expected `mut` or `const` keyword in raw pointer type";
             self.struct_span_err(span, msg)
-                .span_label(span, msg)
-                .help("use `*mut T` or `*const T` as appropriate")
+                .span_suggestions(
```

Comment:
Do we support `span_suggestions` in structured subdiagnostics? If so, then I can put up another commit to convert this.

Reply:
I don't think we do yet, I'll add that to #100717 as a task to do.